### PR TITLE
[FT] CF-61 Add check for ephemeral storage migration

### PR DESCRIPTION
**Pull request by ntymtsiv**
Scenario:
    1. Get volumes on which data was written
    2. Get floating ip address of vm, to which volume attached
    3. Open TCP/22 port for vm's tenant,
    4. Wait until vm accessible via ssh
    5. Check mount point has been migrated with ephemeral storage
    6. Mount volume
    7. Check data on volume is correct

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module devlab.tests.testcases.test_resource_migration
devlab/tests/testcases/test_resource_migration.py:596: 
[W0631(undefined-loop-variable), 
ResourceMigrationTests.test_migrate_cinder_volumes_data.check_mount_point_exists
] Using possibly undefined loop variable 'volume'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
Please add test scenario to pull request description and commit mesage

File: devlab/tests/testcases/test_resource_migration.py [hunk 1]
```diff
@@ -584,6 +584,18 @@ def check_file_valid(filename):
                 msg = "MD5 of file %s before and after migrate is different"
                 raise RuntimeError(msg % filename)
 
+        def check_mount_point_exists(ip, mount_point):
```
Comment (SVilgelm) [inline devlab/tests/testcases/test_resource_migration.py]: add a description (docstring) to this method.

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module devlab.tests.testcases.test_resource_migration
devlab/tests/testcases/test_resource_migration.py:643: 
[W0631(undefined-loop-variable), 
ResourceMigrationTests.test_migrate_cinder_volumes_data.check_mount_point_exists
] Using possibly undefined loop variable 'volume'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
+1, can be merged

**Pull request merged**